### ISSUE TITLE
Issue with rust source code ".rs"

### ISSUE THREAD

**Issue by tripplet** [sbarex/SourceCodeSyntaxHighlight#152]
Hi,
syntax highlighting does not work for .rs files:

```bash
$ mdls -name kMDItemContentType -name kMDItemContentTypeTree main.rs
kMDItemContentType     = "public.rust-source"
kMDItemContentTypeTree = (
    "public.rust-source",
    "public.source-code",
    "public.plain-text",
    "public.text",
    "public.data",
    "public.item",
    "public.content"
)
```

**Comment by sbarex** [comment]
Fixed on release 2.1.10 (58).

**Comment by tripplet** [comment]
Thanks
